tests/test_download_orchestrator.py

```
@@ -50,34 +50,25 @@ def orchestrator(self, mock_config):
         orch = DownloadOrchestrator(mock_config)
-        # Mock the dependencies that are set in __init__
         orch.cache_manager = Mock()
         orch.version_manager = Mock()
         orch.prerelease_manager = Mock()
-        # Mock the downloaders that are created in __init__
-        orch.android_downloader = Mock()
-        orch.android_downloader.download_dir = "/tmp/test"
-        orch.android_downloader.should_download_prerelease.return_value = True
-        orch.android_downloader.update_prerelease_tracking.return_value = True
+        orch.client_app_downloader = Mock()
+        orch.client_app_downloader.download_dir = "/tmp/test"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)